tee-worker/ts-tests/integration-tests/package.json

```diff
@@ -0,0 +1,64 @@
+		"test-identity-direct-invocation:staging": "yarn eslint . && cross-env NODE_ENV=staging mocha --exit --sort -r ts-node/register --loader=ts-node/esm 'identity-direct-invocation.test.ts'",
+		"test-resuming-worker:staging": "yarn eslint . && cross-env NODE_ENV=staging mocha --exit --sort -r ts-node/register --loader=ts-node/esm 'resuming_worker.test.ts'",
+		"test-resuming-worker:local": "yarn eslint . && cross-env NODE_ENV=local mocha --exit --sort -r ts-node/register --loader=ts-node/esm 'resuming_worker.test.ts'",
+		"test-vc:local": "yarn eslint . && cross-env NODE_ENV=local mocha --exit --sort -r ts-node/register --loader=ts-node/esm 'vc.test.ts'",
+		"test-vc:staging": "yarn eslint . && cross-env NODE_ENV=staging mocha --exit --sort -r ts-node/register --loader=ts-node/esm 'vc.test.ts'",
+		"test-batch:local": "yarn eslint . && cross-env NODE_ENV=local mocha --exit --sort -r ts-node/register --loader=ts-node/esm 'batch.test.ts'",
+		"test-batch:staging": "yarn eslint . && cross-env NODE_ENV=staging mocha --exit --sort -r ts-node/register --loader=ts-node/esm 'batch.test.ts'",
+		"test-bulk-vc:local": "yarn eslint . && cross-env NODE_ENV=local mocha --exit --sort -r ts-node/register --loader=ts-node/esm 'bulk_vc.test.ts'",
+		"test-bulk-vc:staging": "yarn eslint . && cross-env NODE_ENV=staging mocha --exit --sort -r ts-node/register --loader=ts-node/esm 'bulk_vc.test.ts'",
+		"test-bulk-identity:local": "yarn eslint . && cross-env NODE_ENV=local mocha --exit --sort -r ts-node/register --loader=ts-node/esm 'bulk_identity.test.ts'",
+		"test-bulk-identity:staging": "yarn eslint . && cross-env NODE_ENV=staging mocha --exit --sort -r ts-node/register --loader=ts-node/esm 'bulk_identity.test.ts'",
+		"di-examples": "yarn eslint . && ts-node examples/direct-invocation/index.ts",
+		"preinstall": "npx only-allow yarn"
+	},
+	"dependencies": {
+		"@noble/ed25519": "^1.7.3",
+		"@polkadot/api": "^10.9.1",
+		"@polkadot/keyring": "^12.2.1",
+		"@polkadot/typegen": "^10.9.1",
+		"@polkadot/types": "^10.9.1",
+		"add": "^2.0.6",
+		"ajv": "^8.12.0",
+		"chai": "^4.3.6",
+		"colors": "^1.4.0",
+		"js-base64": "^3.7.5",
+		"micro-base58": "^0.5.1",
+		"mocha": "^10.1.0",
+		"mocha-steps": "^1.3.0",
+		"scale-ts": "^0.2.11",
+		"websocket-as-promised": "^2.0.1",
+		"ws": "^8.8.1"
+	},
+	"devDependencies": {
+		"@ethersproject/providers": "^5.7.2",
+		"@types/chai": "^4.3.3",
+		"@types/mocha": "^10.0.0",
+		"@types/ws": "^8.5.3",
+		"@typescript-eslint/eslint-plugin": "^5.60.0",
+		"@typescript-eslint/parser": "^5.60.0",
+		"cross-env": "^7.0.3",
+		"dotenv": "^16.0.3",
+		"eslint": "^8.43.0",
+		"ethers": "^5.7.2",
+		"parachain-api": "workspace:*",
+		"prettier": "2.8.1",
+		"sidechain-api": "workspace:*",
+		"ts-node": "^10.9.1",
+		"typescript": "5.0.4"
+	},
+	"compilerOptions": {
+		"allowSyntheticDefaultImports": true,
+		"esModuleInterop": true
+	},
```

Comment:
Are these still needed? (i.e. they should be controlled by `tsconfig.json` AFAIK)

Reply:
fixed!